lxd/db/backups.go

```diff
@@ -282,6 +282,46 @@ func (c *Cluster) GetExpiredInstanceBackups() ([]InstanceBackup, error) {
 	return result, nil
 }
 
+// GetExpiredStorageVolumeBackups returns a list of expired storage volume backups.
+func (c *Cluster) GetExpiredStorageVolumeBackups() ([]StoragePoolVolumeBackup, error) {
+	var backups []StoragePoolVolumeBackup
+
+	q := `SELECT storage_volumes_backups.name, storage_volumes_backups.expiry_date, storage_volumes_backups.storage_volume_id FROM storage_volumes_backups`
```

Comment:
Does this need to filter by volume node ID? 
How do we deal with backup/volume expiry for volumes on remote storage pools?

Reply:
> Does this need to filter by volume node ID?

Possibly.

>  How do we deal with backup/volume expiry for volumes on remote storage pools?

Backups aren't stored on the storage pools but in `${VarPath}/backups/custom/<poolName>/<volumeName>`.

Reply:
> Backups aren't stored on the storage pools but in `${VarPath}/backups/custom/<poolName>/<volumeName>`.

Ah OK makes sense.